python/az/aro/azext_aro/custom.py

```diff
@@ -59,7 +59,7 @@ def aro_create(cmd,  # pylint: disable=too-many-locals
 
     aad = AADManager(cmd.cli_ctx)
     if client_id is None:
-        app, client_secret = aad.create_application('aro-%s' % random_id)
+        app, client_secret = aad.create_application(cluster_resource_group or 'aro-' + random_id)
```

Comment:
Nit: on line 96 we do the same. Is it worth writing cluster RG into a variable (eg `cluster_resource_group = cluster_resource_group or 'aro-' + random_id`) and use it everywhere where we need it?

Reply:
> This looks good to me, but could please add more context why we need it? Is it for purging?

yes, as currently there is no logical connection between name of app and cluster.